smaug/python/ops/array_ops.py

```diff
@@ -344,31 +344,27 @@ def check_and_add_layout_transform(name, op, input_tensors):
       input_tensors[i] = reorder(input_tensors[i], expected_layoutset.layouts)
   return input_tensors
 
-def padding(input_tensor, padder, name="padding"):
+def padding(input_tensor, padding_size, name="padding"):
   """Construct a tensor by padding a given tensor.
 
   Args:
     input_tensor: Input tensor.
-    padder: A int value that represents the padding dimension
+    padding_size: A list in the format of {dim0_begin, dim0_end, dim1_begin, dim1_end, ...} that 
```

Comment:
Add a note that the order of dimensions must align with the data layout of input_tensor.

Also, ensure that this line doesn't exceed 80 characters in length.